charger/ocpp.go

```diff
@@ -48,8 +47,8 @@ func NewOCPPFromConfig(other map[string]interface{}) (api.Charger, error) {
 		MeterInterval  time.Duration
 		MeterValues    string
 		ConnectTimeout time.Duration // Initial Timeout
```

Comment:
Das hier sollten wir wohl auch besser umbenennen, da dies mit dem festen OCPP-Begriff `ConnectionTimeOut` Verwechslungsgefahr birgt aber etwas ganz anderes meint:

_Interval *from beginning of status: 'Preparing' until incipient Transaction is automatically canceled, due to failure of EV driver to (correctly) insert the charging cable connector(s) into the appropriate socket(s). The Charge Point SHALL go back to the original state, probably: 'Available'._

Reply:
Ich möchte zu zwei existierenden Timeouts jetzt keinen weiteren einführen der noch neue Verwirrung stiftet. Immer dran denken: dass müssen die Anwender alles nachvollziehen...

Reply:
Das will ich ja auch nicht.
Es ging nur um die Namensgebung, die speziell im OCPP-Umfeld halt etwas verwirrend sein könnte.